modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/SAMLARTRESOLVETestCase.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Update the license.

Reply:
Fixed https://github.com/wso2/product-is/pull/8590/commits/db6519d17a39bc68aa08982b911eb1fe830fe1a9